### PR TITLE
Hw10

### PR DESCRIPTION
**Pull request by o-yelkoff**
Hi, 
please review my homework#10

### CODE REVIEW

File: src/test/java/gitHubTest/APIFirstTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchema;
+import static org.hamcrest.Matchers.*;
+
+public class APIFirstTest extends BaseAPITest {
+
+    private final String API_KEY = "ac3ae531e9fea0c8f439cd51bc5b015e";
```
Comment (DarynaHorobei97) [inline src/test/java/gitHubTest/APIFirstTest.java]: why final? this value will be changed due to expiring

**Review by DarynaHorobei97 — COMMENTED** [review]
Good job but also please add test with a pojo usage

**Comment by o-yelkoff** [comment]
TEST1

**Review by o-yelkoff — COMMENTED** [review]
test

**Pull request merged**